src/merkle/path.rs

```diff
@@ -28,6 +28,11 @@ impl MerklePath {
         self.nodes.len() as u8
     }
 
+    /// Returns a reference to the [MerklePath]'s nodes.
+    pub fn nodes(&self) -> &Vec<RpoDigest> {
+        &self.nodes
+    }
```

Comment:
I would prefer to stick to the guidelines:

https://rust-lang.github.io/api-guidelines/predictability.html#only-smart-pointers-implement-deref-and-derefmut-c-deref
https://doc.rust-lang.org/std/ops/trait.Deref.html

This is not a smart pointer and should not implement `Deref`.

Reply:
Yeah - makes sense. Let's remove the `Deref` implementation.

I'd still probably prefer though to return `&[RpoDigest]` instead of `&Vec[RpoDigest]`.

Reply:
changed to return a slice